apps/ecs/index.js

```diff
@@ -22,7 +23,45 @@ module.exports = {
 
     },
     '/already-employed': {
-
-    }
+      fields: ['person-work-for-you'],
+      forks: [
+        {
+          target: '/when-started',
+          condition: {
+            field: 'person-work-for-you',
+            value: 'yes'
+          }
+        },
+        {
+          target: '/digital-right-to-work-service',
+          condition: {
+            field: 'person-work-for-you',
+            value: 'no'
+          }
+        }
+      ]
+    },
+    '/when-started': {
+      fields: ['start-work-date'],
+      forks: [
+        {
+          target: '/tupe',
+          condition: req => req.sessionModel.get('start-work-date') < '2008-02-29'
```

Comment:
`'2008-02-29' ` - since we need to have this date check in many steps.  i think it would be better to have this date in config file and use it further.